using-kite-properties.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Using Kite Properties
+---
+
+Kite gives you the option of changing settings of existing system properties to enhance performance. You also have the option of creating custom properties.
+
+## Setting Kite Properties
+
+There are several dataset properties that affect how files are written. These settings are especially relevant for Parquet because it buffers records in memory. 
+
+`kite.write.cache-size` controls the number of files kept open by an HDFS or Hive dataset writer.
```

Comment:
I don't mind adding headings, and I've done so. Generally, it's non-standard to create subheadings when there is only one. The reason I put the two properties together is that they act in concert, and they are the only two properties we're documenting, at the moment. This will settle out when we add descriptions of additional properties.